ash/src/entry.rs

```diff
@@ -149,6 +149,23 @@ impl Entry {
         let entry_fn_1_0 = vk::EntryFnV1_0::load(load_fn);
         let entry_fn_1_1 = vk::EntryFnV1_1::load(load_fn);
```

Comment:
Inline these like in `Device`?

Reply:
rust complains about lifetimes if I do

Reply:
We'd have to `Copy` `let get_instance_proc_addr = static_fn.get_instance_proc_addr;` in that case.

Reply:
It was trivial to fix by making the closure `move`